File: app/src/main/java/com/BlizzardArmory/warcraft/CharacterSummary/RaidProgression.java
package com.BlizzardArmory.warcraft.CharacterSummary;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RaidProgression {

    @SerializedName("href")
    @Expose
    private String href;

    public String getHref() {
        return href;
    }

    public void setHref(String href) {
        this.href = href;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/charactersummary/RaidProgression.kt
package com.BlizzardArmory.model.warcraft.charactersummary

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Raid progression.
 */
@Keep
data class RaidProgression(

    @SerializedName("href")
    var href: String

)